function-aws-api-proxy/src/main/java/io/micronaut/function/aws/proxy/cookie/ObjectUtil.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2014 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License, version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at:
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.micronaut.function.aws.proxy.cookie;
+
+import io.micronaut.core.annotation.Internal;
+
+/**
+ * A grab-bag of useful utility methods.
+ *
+ * Forked from Netty.
+ */
+@Internal
+public final class ObjectUtil {
```

Comment:
remove this class it is not necessary we already have methods that do this or can be replaced by `Objects.requireNonNull`

Reply:
resolved via https://github.com/micronaut-projects/micronaut-aws/pull/1510/commits/95806e27850a7e21ec52ad2db99aa1221fb5a839